docs/content/manual/volumes/prevent-attached-volume-to-other-vm.md

```diff
@@ -0,0 +1,34 @@
+---
+title: A volume can't be attached to another VM (Yaml)
+---
+
+* Related issues: [#5383](https://github.com/harvester/harvester/issues/5383) [ENHANCEMENT] Refactor harvesterhci.io/owned-by annotation on PVC
+
+## Category: 
```

Comment:
Redundant colon?

Reply:
Thanks for the check. Remove the redundant colon.